src/VsExtension/NuGetPackage.cs

```diff
@@ -62,7 +62,7 @@ namespace NuGetVSExtension
     public sealed class NuGetPackage : Package, IVsPackageExtensionProvider, IVsPersistSolutionOpts
     {
         // It is displayed in the Help - About box of Visual Studio
-        public const string ProductVersion = "3.1.2";
+        public const string ProductVersion = "3.2";
```

Comment:
It doesn't matter because the assembly version is changed by the build script on the CI with the build version.